dockertest/subtest.py

```diff
@@ -429,13 +458,61 @@ def try_all_stages(self, name, subsubtest):
                               self.exception_info["error_source"],
                               detail)
             exc_info = self.exception_info["exc_info"]
+            # Ignore known problems in specific docker builds
+            if self.is_known_failure(name):
+                self.logwarning("Treating %s subsubtest failure as PASS", name)
+                self.final_subsubtests.add(name)
             # Treat N/A as passed, even though this may hide obscure failures
             if isinstance(detail, TestNAError):
```

Comment:
> two warning messages.

That's what I was wondering about, yes the outcome is the same, except for maybe a confused developer at the other end.

Let's just leave this way for now, it's really not a big deal.